index.html

```diff
@@ -1,230 +1,336 @@
+        </div>
+        <div class="container">
+          <div class="icon-circle">
+            <img src="Images/icon-skills.png" alt="Skills-icon" />
+          </div>
+          <h3 class="about-title">Skills</h3>
+          <div class="card-buttons">
+            <ul>
+              <li>Codekit</li>
+              <li>Github</li>
+              <li>Codepen</li>
+              <li>Gitlab</li>
+              <li>Terminal</li>
+            </ul>
+          </div>
         </div>
       </div>
-      <div class="container">
-        <div class="icon-circle">
-          <img src="Images/icon-skills.png" alt="Skills-icon" />
+    </div>
+    <div class="contact-form">
+      <nav class="text-layout">
+          <h4 class="form-text">I'm always interested in hearing about new projects, so if you'd like to chat please get in touch.</h4>
+      </nav>
+      <form action="https://formspree.io/f/mbjbdrnb" class="form-data">
+        <fieldset>
+          <label><input class="label" type="text" name="first-name" placeholder="Full name" required /></label>
+          <label><input class="label" type="email" name="email" placeholder="Email address" required /></label>
+          <label>
+            <textarea class="bio" rows="3" cols="30" placeholder="Enter text here"></textarea>
+          </label>
+        </fieldset>
+        <div class="touch-box">
+          <button class="get-in-touch" type="button">Get in touch</button>
         </div>
-        <h3 class="about-title"> Skills </h3>
-        <div class="card-buttons">
+        <div class="lower-logo">
           <ul>
-            <li>Codekit</li>
-            <li>Github</li>
-            <li>Codepen</li>
-            <li>Gitlab</li>
-            <li>Terminal</li>
+            <li>
+              <a href="#"
+                ><img
+                  class="img"
+                  src="Images/twittertwitter.png"
+                  alt="Twitter logo"
+              /></a>
+            </li>
+            <li>
+              <a href="#"
+                ><img class="img" src="Images/Linkedin.png" alt="LinkedIn logo"
+              /></a>
+            </li>
+            <li>
+              <a href="#"
+                ><img
+                  class="img"
+                  src="Images/mediummedium.png"
+                  alt=" Medium logo"
+              /></a>
+            </li>
+            <li>
+              <a href="#"
+                ><img
+                  class="img"
+                  src="Images/githubgithub.png"
+                  alt="Github logo"
+              /></a>
+            </li>
+            <li>
+              <a href="#"
+                ><img
+                  class="img"
+                  src="Images/angellistangelist.png"
+                  alt="Angellist logo"
+              /></a>
+            </li>
           </ul>
         </div>
-      </div>
+        <img class="illustration-contact" src="Images/illustration-contact-form.png" alt="illustration-contact-form">
```

Comment:
- [ ] kindly move this code outside of the form tag , since the form should only have the inputs of data that we need to send to formspree